sqlpage/templates/carousel.handlebars

```diff
@@ -0,0 +1,40 @@
+<div class="card my-2 col-md-{{default width 12}}{{#if center}} mx-auto{{/if}}">
+    <div class="card-body">
+        {{#if title}}
+            <div class="d-flex align-items-center">
+                <div class="subheader">{{title}}</div>
+            </div>
+        {{/if}}
+        <div id="{{name}}" class="carousel slide{{#if fade}} carousel-fade{{/if}}" data-bs-ride="carousel">
+            <div class="carousel-indicators{{#if indicators}} carousel-indicators-{{indicators}}{{/if}}{{#if vertical}} carousel-indicators-vertical{{/if}}">
+                {{#each_row}}
+                    <button type="button" data-bs-target="#{{../name}}" data-bs-slide-to="{{@row_index}}" {{#if (eq @row_index 0)}}class="active"{{/if}}></button>
+                    {{#delay}}
+                        {{flush_delayed}}
+                        <div class="carousel-item {{#if (eq @row_index 0)}}active{{/if}}">
+                            <img class="d-block w-100" alt="{{image}}" src="{{image}}" />
+                            {{#if title}}
+                            <div class="carousel-caption d-none d-md-block">
+                                <h5>{{title}}</h5>
+                                {{#if contents}}<p>{{contents}}</p>{{/if}}
```

Comment:
Can we add the possibility to render markdown descriptions ?

Like other components, with `description` and `description_md`